src/sample/Main.java

```diff
@@ -16,7 +16,7 @@
 public class Main extends Application {
 
     public static void main(String[] args) {
-        String inputFile = "examples/example.in";
+        String inputFile = "examples/board_8x8_hard.slow";
```

Comment:
I think, we shouldn't commit changes in Main.java that are used only for testing/running app for debug.